packages/realm/src/Types.ts

```diff
@@ -18,6 +18,8 @@
 
 import { BSON, Dictionary, List, RealmSet, Results } from "./internal";
 
+export type Mixed = unknown;
```

Comment:
@kraenhansen before I was importing `Mixed` from `.` in `error.ts`, but you correctly suggested to import it from `./internal`, but it's not available there. So I added it here for now, but I am not sure of what's a good place for that. Directly `internal`?

Reply:
When importing it in `error.ts`, try importing `Types` from `./internal` instead. Was `Mixed` not available on `Types`?

Reply:
I can do that, but then I'll need to use it as `Types.Mixed`. Does it make sense?

Reply:
Discussed with @kraenhansen , and I will use `PrimaryKey` as the type here, as it makes sense